framework/src/modules/interoperability/mainchain/types.ts

```diff
@@ -0,0 +1,7 @@
+export enum ForwardResult {
+	COULD_NOT_TRANSFER_FWD_FEE,
+	SUCCESS,
+	INVALID_CCM,
+	NO_ACTIVE_RECV_CHAIN,
+	INFORM_SIDECHAIN_TERMINATION,
```

Comment:
maybe use full name instead?
`COULD_NOT_TRANSFER_FORWARD_FEE`
`INACTIVE_RECEIVING_CHAIN`

Reply:
Done. I thought we introduced abbrs to framework, otherwise I don't prefer abbrs 🙂

Reply:
I thought atleast within an enum maybe we can have consistency :)